.github/PULL_REQUEST_TEMPLATE/mod_recipe.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Modified recipe and diagnostic
+about: Modification to an already existing recipe and diagnostic
+title: ''
+labels: 'diagnostic'
+assignees: ''
+
+---
+
+Please discuss your idea for modifications to a diagnostic or recipe with the development team and the author of the diagnostic before getting started, to avoid disappointment later. A good way to do this is to open an issue on GitHub. This is also a good way to get help.
+
+---
+
+**Acceptance Criteria**
+
+* All tests must pass (see the CI tests at the bottom of the discussion and click on "Details" to see why a test fails
+
+---
+
+**Tasks**
+
+- [ ] Add any additional dependencies needed for the diagnostic script to setup.py
```

Comment:
adding package dependencies is a very tricky one - that needs to be discussed in advance and we need to make sure there is absolutely no way we can not **not** add that dependency, I'd add a line saying something like this